openfe/utils/visualization_3D.py

```diff
@@ -106,6 +106,39 @@ def _add_spheres(view:py3Dmol.view, mol1:Chem.Mol, mol2:Chem.Mol, mapping:Dict[i
             }
         )
 
+@requires_package("py3Dmol")
+def show_ComponentCoords(mols: Union[Component, Iterable[Component]],
+                     style: Optional[str] ="stick",
+                     ) -> py3Dmol.view:
+    """this function can be used to visualize multiple component coordinates in one view.
```

Comment:
what happens if you pass a solvent component?

Reply:
Uh, they don't have an rdkit underneath, right?! 
hm... then I guess I need to be more strict in the typing. (ProteinComponent or SmallMoleculeComponent)